components/Statistics/StatisticsActivityTagsReport.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import { FormattedMessage } from 'react-intl'
+import { LoadSpinner } from '../Loading'
+import { PieChart, Pie, Tooltip, Legend, Cell, ResponsiveContainer } from 'recharts'
+import styled from 'styled-components'
+import { useStatisticsAPI } from '../../lib/statistics/statisticsHooks'
+
+const COLORS = ['#0088FE', '#00C49F', '#FFBB28', '#FF8042', '#653CAD']
```

Comment:
get walt to generate a suitable palette.

Reply:
Ran a quick colorblindless check - the colours are fine imo